index.html

```diff
@@ -62,14 +89,22 @@ <h1>
             </a>
           </li>
         </ul>
+        <div class="disabled-button">
+          <button type="button">
+            <img src="Imgs/Disabled.svg" alt="disabled button" />
+          </button>
+        </div>
       </div>
     </section>
 
     <!-- My recent Works Section -->
 
     <section>
-      <div class="jobs" id="jobs-id">
+      <div class="recent-job">
         <h1>My Recent Works</h1>
+      </div>
```

Comment:
- [ ] The green bar is not visible in the desktop design as observed in the Figma Design. Kindly remove it

Your work
![image](https://user-images.githubusercontent.com/84156714/235983139-54ace253-836a-4492-bb5c-d6e472402199.png)

Figma Design
![image](https://user-images.githubusercontent.com/84156714/235983266-667e3770-ab27-4123-a98a-4e5f7d740366.png)